cfgov/unprocessed/js/modules/transition/AlphaTransition.js

```diff
@@ -43,28 +45,37 @@ function AlphaTransition( element ) {
   }
 
   /**
-   * Remove all transition classes.
+   * Remove all transition classes, if transition is initialized.
+   * @returns {boolean}
+   *   True, if the element's CSS classes were touched, false otherwise.
    */
   function remove() {
-    _baseTransition.remove();
+    return _baseTransition.remove();
   }
 
   /**
    * Add a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOn() {
-    _baseTransition.animateOn();
+    return _baseTransition.animateOn();
   }
 
   /**
    * Remove a "transition-duration: 0" utility CSS class.
+   * @returns {boolean}
+   *   Whether the element's CSS classes where touched or not.
+   *   False means the transition has not been initialized.
    */
   function animateOff() {
-    _baseTransition.animateOff();
+    return _baseTransition.animateOff();
```

Comment:
No, it's still relevant. `BaseTransition` would just follow the pattern of `EventObserver`.